mvrx/src/main/kotlin/com/airbnb/mvrx/StatefulView.kt

```diff
@@ -0,0 +1,12 @@
+package com.airbnb.mvrx
+
+/**
+ * Implement this in a custom view to expose [fragmentViewModel] and [activityViewModel] in a custom view. Similar to usage in Fragments,
+ * this will create a view-lifecycle-aware subscription.
+ * For [fragmentViewModel], this will walk up the view hierarchy until it finds a View that is the root view for a Fragment.
+ * [StatefulView] uses [onInvalidate] instead of `invalidate()` because `invalidate()` is already a function on View. It is also
+ * not called [MvRxView] because the name was already taken.
+ */
+interface StatefulView {
```

Comment:
kind of a shame that we can't better combine this with the existing `MvRxView` - I wonder if that should be named `MvRxFragment`? it would also be nice if we could rename `MvRxView#invalidate` to match this.

honestly I'm kind of ok with breaking changes at this point to keep this more in sync

Reply:
@elihart I'm open to have a full discussion about this. The original idea behind MvRxView is that you could use it for other things like conductor or other custom architectures. However, the full functionality of MvRxView is explicitly not needed here which is now awkward.

Renaming invalidate to onInvalidate everywhere is possible but would require a lot of updates but each one would be a simple copy/replace. What if we renamed it `render` to make it more like React?

Reply:
@gpeal I would be ok with a rename to `render` I think. or is there a keyword that Compose is using that we could match now?

thoughts @BenSchwab